app/src/main/java/team7/voluntime/Fragments/Common/UserProfileFragment.java

```diff
@@ -45,4 +151,18 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
     }
 
+    @OnClick(R.id.edit)
+    public void onClick(View arg0) {
+        Toast.makeText(getContext(), "Function Not Complete", Toast.LENGTH_LONG).show();
+
+    }
+
+
+    public String getType() {
+        String type = MainActivity.getAccountType();
+        if (type.equals("Volunteer")) return "Volunteers";
+        else return "Charities";
```

Comment:
for conditionals, just to keep it consistent across the app, can we do:

```
if (type.equals("Volunteer")) {
    return "Volunteers";
} else {
     return "Charities";
}
```
Even though its only 1 line and yeh you dont need the curly bois, it makes it consistent to add them and imo slightly more readable (this is subjective though).